helpdesk_mgmt_stage_validation/models/helpdesk_ticket.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, models
+from odoo.exceptions import ValidationError
+
+
+class HelpdeskTicket(models.Model):
+    _inherit = "helpdesk.ticket"
+
+    def _check_ticket_has_empty_fields(self, stage):
```

Comment:
No need to pass stage as argument, you can have it with `self.stage_id`.

```suggestion
    def _check_ticket_has_empty_fields(self):
```